solutions/secure-regional-bucket/README.md

```diff
@@ -0,0 +1,10 @@
+# Cloud Object Storage for IBM Cloud - Financial Services Cloud solution
+
+This architecture creates an Cloud Object Storage bucket on IBM Cloud® and supports provisioning the following resources:
+
+- A Regional COS Bucket
+- Context based restriction rules for the instance and buckets
```

Comment:
- `an Cloud Object Storage bucket` -> a Cloud Object Storage(COS) bucket
- `A Regional COS Bucket` -> A regional COS bucket
- instance can be removed